Makefile

```
@@ -117,19 +117,20 @@ ${NORMALIZATION_GRPC}: ${NORMALIZATION_PROTO} ${PROTOC_GEN_GRPC_SWIFT}
 .PHONY:
 generate-normalization: ${NORMALIZATION_PB} ${NORMALIZATION_GRPC}
 
+SERIALIZATION_GRPC_REFLECTION=Tests/GRPCTests/Codegen/Serialization/echo.grpc.reflection.txt
 SERIALIZATION_PATH=Tests/GRPCTests/Codegen/Serialization/
 
-# For serialization we'll set the ReflectionService option to true.
-${SERIALIZATION_GRPC}: ${ECHO_PROTO} ${PROTOC_GEN_GRPC_SWIFT}
+# For serialization we'll set the ReflectionData option to true.
+${SERIALIZATION_GRPC_REFLECTION}: ${ECHO_PROTO} ${PROTOC_GEN_GRPC_SWIFT}
 	protoc $< \
 		--proto_path=$(dir $<) \
 		--plugin=${PROTOC_GEN_GRPC_SWIFT} \
-		--grpc-swift_opt=ReflectionService=true \
+		--grpc-swift_opt=Client=false,Server=false,ReflectionData=true \
 		--grpc-swift_out=$(dir ${SERIALIZATION_PATH})
```

There's no need to define the `SERIALIZATION_PATH` above, you can get the directory of the file we want to generate:

```suggestion
		--grpc-swift_out=$(dir ${SERIALIZATION_GRPC_REFLECTION})
```